### ISSUE TITLE
for single-use algorithm, document whether the issuer and holder algorithms may be different

### ISSUE THREAD

**Issue by dwaite** [ietf-wg-jose/json-web-proof#78]
From @OR13, https://mailarchive.ietf.org/arch/msg/jose/GkL3I9hEoKjaIDCgno_rYqQsSP0/ :

> An example with a mixed algorithm for issue and holder would be nice here,
or a comment about it being forbidden... for example issuer signing with
ES256 and holder presenting with EdDSA.

**Comment by selfissued** [comment]
Clearly the issuer and holder algorithms have to be able to be different.  I'll review the specs with this in mind.

**Comment by dwaite** [comment]
I believe this work has been done, @selfissued are you ok with this being closed?

**Comment by selfissued** [comment]
Let's consider this together with #57.  On the surface of it, it seems that we have to to address that one to address this one.

**Comment by dwaite** [comment]
I do not think this is the same as #57 / #59 . We just need to make this clear for algorithms in general

**Comment by selfissued** [comment]
We talked about this on the 11-Jun-25 editors' call.  One way to handle this would be to define a `presentation_alg` header parameter.  Another way would be to put the `alg` in the `presentation_alg`.

Of course, for some ZKP algorithms such as BBS, you don't get to choose a separate presentation algorithm.
